### ISSUE TITLE
[BUG] Eerie Entities Wisps spawn in ludicrous numbers

### ISSUE THREAD

**Issue by anabsolutesloth** [Minecraft-Eternal/MC-Eternal-1.12#292]
**Describe the bug**
Eerie Entities wisps overspawn a lot

**Version of MC Eternal**
1.4.4

**To Reproduce**
1. Go to Erebus
2. Explore
3. suffer

**Expected behavior**
No suffer

**Screenshots**
![image](https://user-images.githubusercontent.com/63694982/126742194-eeedaafb-84b9-4a1a-97d7-2434c8cb29e2.png)

**Additional context**
This is mostly a reminder for myself to find a fix

**Comment by criscky** [comment]
use incontrol to limit that?

**Comment by criscky** [comment]
adapt this code
```
  {
    "dimension": 28885,
	"mob": "artifacts:mimic",
    "mincount": {
      "amount": 5,
	  "mob": "artifacts:mimic"
    },
    "result": "deny"
  }
```

**Comment by anabsolutesloth** [comment]
should be fixed as of like mce 1.5, but definitely is in 1.6+
